code/modules/unit_tests/resist.dm

```diff
@@ -12,6 +12,11 @@
 	// Stop, drop, and roll has a sleep call. This would delay the test, and is not necessary.
 	call_async(human, /mob/living/verb/resist)
 
+	//since resist() is a verb that possibly queues its actual execution for the next tick, we need to make the subsystem that handles the delayed execution process
+	//the callback. either that or sleep ourselves and see if it ran.
+	if(human.fire_stacks > 5)
```

Comment:
why the check?

Reply:
because flushing the queue if the callback we wanted wasnt in it still has a non zero chance of mutating the world and causing the test to fail, but now that i think of it we want that anyways